src/test/java/spoon/test/filters/FilterTest.java

```diff
@@ -883,16 +884,16 @@ public void testClassCastExceptionOnForEach() throws Exception {
 		{
 			Context context = new Context();
 			//contract: if the query produces elements which cannot be cast to forEach consumer, then they are ignored
-			launcher.getFactory().Package().getRootPackage().filterChildren(f->{return true;}).forEach((CtType t)->{
+			launcher.getFactory().Package().getRootPackage().filterChildren(null).forEach((CtType t)->{
```

Comment:
why do you need to change this contract?

Reply:
it is not change. It is just shorter way how write the same: "no filter = return all children"